libminifi/include/core/Core.h

```diff
@@ -80,13 +81,16 @@ static inline std::string getClassName() {
   std::free(b);
   return name;
 #else
-  std::string adjusted_name = typeid(T).name();
-  // can probably skip class  manually for slightly higher performance
-  const std::string clazz = "class ";
-  auto haz_clazz = adjusted_name.find(clazz);
-  if (haz_clazz == 0)
-    adjusted_name = adjusted_name.substr(clazz.length(), adjusted_name.length() - clazz.length());
-  return adjusted_name;
+  std::string_view name = typeid(T).name();
+  const std::string_view class_prefix = "class ";
+  const std::string_view struct_prefix = "struct ";
+
+  if (name.find(class_prefix) == 0) {
```

Comment:
Since C++20 we can use [starts_with](https://en.cppreference.com/w/cpp/string/basic_string_view/starts_with) which is a bit more readable

Reply:
We also have a `startsWith` function in the StringUtils, maybe in the future in a separate jira ticket that can be removed in favor of the c++20 variant.

Reply:
last time I tried to use `std::string_view::starts_with`, I recall that some of our platforms did not support it, for now added `StringUtils::startsWith` so we could bulk replace it as @lordgamez suggested